### PR TITLE
saucelabs-espresso-app-automate 0.1.5

### PR DESCRIPTION
**Pull request by moatazeldebsy**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by moatazeldebsy** [comment]
Sorry there was a problem in the script, not the updated one and edit the description for the device argument

File: steps/saucelabs-espresso-app-automate/0.1.5/step.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      A list of available device and OS combinations can be found [here](https://saucelabs.com/platform/supported-browsers-devices)
+    is_expand: true
+    is_required: true
+    summary: The desired test device
+    title: Device Name
+  saucelabs_device_name: ""
+- opts:
+    description: |
+      Set this parameter if you want to specify a Sauce Labs datacenter Options: "us-west-1" , "eu-central-1"
+    is_expand: false
```
Comment (koral--) [inline steps/saucelabs-espresso-app-automate/0.1.5/step.yml]: Why this parameter is not expandable?

Comment (moatazeldebsy) [reply steps/saucelabs-espresso-app-automate/0.1.5/step.yml]: @koral-- because it will include only one option `us-west-1` or  `eu-central-1`

Comment (koral--) [reply steps/saucelabs-espresso-app-automate/0.1.5/step.yml]: OK, so it should contain `value_options`.
https://devcenter.bitrise.io/bitrise-cli/step-inputs/

Comment (moatazeldebsy) [reply steps/saucelabs-espresso-app-automate/0.1.5/step.yml]: Shall I close this PR a create a new one with new tag or the same

Comment (trapacska) [reply steps/saucelabs-espresso-app-automate/0.1.5/step.yml]: @moatazeldebsy yes please, it would end up in a more easy way to configure for the users! 🙂

Comment (moatazeldebsy) [reply steps/saucelabs-espresso-app-automate/0.1.5/step.yml]: sure will do

**Pull request closed without merging**